.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: node_js
+node_js:
+  - "7"
+before_script:
+  - export CHROME_BIN=chromium-browser
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
+  - npm install -g gulp 
+script: 
+  - npm test

```

Comment:
Could you please fix the line endings? (I can't seem to do it when editing on the Website)